src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -186,39 +186,41 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.ColonColonToken))
+            if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
             {
-                text = "::_CSharpKeyword";
+                text = "$_CSharpKeyword";
```

Comment:
can you provide the links these now go to.

Reply:
The `$_CSharpKeyword` should go to this:

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
and is that where `$_CSharpKeyword` does redirect to?  Can you give me the links these produce directly, so i can validate they end up landing on an appropriate page?  Thanks!

Reply:
https://msdn.microsoft.com/query/dev16.query?appId=Dev16IDEF1&l=EN-US&k=k($_CSharpKeyword);k(DevLang-csharp)&rd=true

Reply:
That link takes me to https://docs.microsoft.com/en-us/.

Reply:
It was redirecting correctly at the time I commented with it. For me, it now redirects nowhere. Just keeps loading without doing anything. Probably it's temporary issue with that service? Actually, **ALL** the F1 thing uses that link.

Reply:
I can confirm that at the time of writing this, the whole service isn't working correctly. Pressing F1 on **anything**, for example await keyword, either take much loading time without redirecting anywhere, or redirects to the home docs page as you stated @CyrusNajmabadi.

Let's wait a little bit and check if the service will back to normal. Otherwise, I don't know what we should do.

But anyways, the PR just sets the f1 keyword for string interpolation and it matches the f1_keywords metadata in dotnet/docs. So, it should work whenever the service back to normal.

Reply:
Also, adding @davidwengier as he touched this area before (https://github.com/dotnet/docs/pull/19700#issuecomment-664838915)

Reply:
It actually redirects to "/Areas/Epx/Content/500.aspx?aspxerrorpath=/query/dev16.query" first, which in turn redirected to the home docs.
So, it's an internal server error.

Reply:
The query service seems to be pretty flaky at the moment, not sure what the deal is with that. The link above (msdn.microsoft.com/query/dev16.query?appId=Dev16IDEF1&l=EN-US&k=k($_CSharpKeyword);k(DevLang-csharp)&rd=true) worked for me, and looks how I suspect.